docs/source/getting-started/installation.rst

```diff
@@ -54,7 +64,7 @@ or install from source:
 .. _cloud-account-setup:
 
 Cloud account setup
--------------------
+===================
```

Comment:
```suggestion
-------------------
```
without "Docker Installation" becoming a header, the original ToC / nesting level seems clearer :)